torchvision/models/quantization/googlenet.py

```diff
@@ -18,7 +20,13 @@
 }
 
 
-def googlenet(pretrained=False, progress=True, quantize=False, **kwargs):
+def googlenet(
+    pretrained: bool = False,
+    progress: bool = True,
+    quantize: bool = False,
+    **kwargs: Any
+):
```

Comment:
missing return type: `QuantizableGoogLeNet`

Reply:
I cannot add this since `QuantizableGoogLeNet` is declared later in the file. (Same for Inception)
It would need a bit of overwrite over git history.
If all are fine I will proceed.